app/repository/PropertyDetailsMongoRepository.scala

```diff
@@ -108,28 +111,35 @@ class PropertyDetailsReactiveMongoRepository(mongo: () => DB, val metrics: Servi
 
     foundPropertyDetails flatMap { propertyDetails =>
```

Comment:
`val foundPropertyDetails...` needs to be wrapped also otherwise will have lost MDC by the time do the flatmap.